### PR TITLE
company_codes field added.

### PR DESCRIPTION
**Pull request by eozan**
SD-4968

### CODE REVIEW

File: superdesk/metadata/item.py [hunk 1]
```diff
@@ -155,6 +155,17 @@
             }
         }
     },
+    'company_codes': {
```
Comment (mdhaman) [inline superdesk/metadata/item.py]: Can you remove `company_codes` definition from https://github.com/superdesk/superdesk-core/blob/master/apps/archive/common.py#L138?

Comment (eozan) [inline superdesk/metadata/item.py]: done.

**Comment by ioanpocol** [comment]
:+1:

**Comment by petrjasek** [comment]
:+1:

**Pull request merged**